onnx_caffe2/backend.py

```diff
@@ -673,6 +673,10 @@ def onnx_graph_to_caffe2_net(cls, graph_def, device="CPU"):
 
             init_net.op.extend([op_def])
 
+        # Set the device option for the init_net and predict_net.
+        init_net.device_option.CopyFrom(device_option)
+        predict_net.device_option.CopyFrom(device_option)
```

Comment:
if you add this here, can you delete it on line 512/515?

Reply:
Sure. Done.